x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/hooks/use_endpoint_action_items.tsx

```diff
@@ -62,7 +67,7 @@ export const useEndpointActionItems = (
 
       const isolationActions = [];
 
-      if (isIsolated) {
+      if (isIsolated && isolationSupported) {
```

Comment:
What are the (rather odd) chances that someone "isolated" an unsupported host via other means and now they can't release it through the UI?

Is this check necessary? if a host can't be isolated, certainly they should never reach the first part of the condition

Reply:
In this case, if Isolation isn't supported, the Endpoint would never be able to actually isolate, even if we sent it an action.  The Endpoint would ignore it because it doesn't recognize the action request.  This will be true for old Endpoints and for Linux Endpoints in `7.14`.

Reply:
I'm thinking that regardless if it is supported or not, if we know the host is isolated, then we should allow `unisolate`. Hopefully this will never happen, but to be safe, I would revert this change here